src/shell/mpir/rangelist.c

```diff
@@ -25,16 +25,20 @@
  *   which is delta encoded from the previous entry (or 0 if this is the
```

Comment:
In "shell: mpir: compress repeating ranges in rangelist" commit message:  s/commoon/common/
Also is that space in one of the ranges supposed to be there?